packages/backend/src/controllers/slackController.ts

```diff
@@ -39,6 +40,7 @@ export class SlackController extends BaseController {
     @OperationId('getSlackChannels')
     async get(
         @Request() req: express.Request,
+        @Query() search?: string,
```

Comment:
You need to `yarn generate-api` to start using this parameter, otherwise tsoa will ignore this argument on the query (I was getting undefined all the time)

Reply:
![image](https://github.com/user-attachments/assets/d0c160d8-0986-4d4c-b3db-5581d5d0194b)